config.js

```diff
@@ -221,9 +221,14 @@ var config = {
     // Specifies whether the raised hand will hide when someone becomes a dominant speaker or not
     // disableRemoveRaisedHandOnFocus: false,
 
+    // Specifies whether the speaker stats is enable or not
+    // speakerStats.disabled
```

Comment:
Please document the whole object and the available options with valid JS syntax.

Reply:
THis comment is now redundant.